secure.go

```diff
@@ -192,10 +195,14 @@ func (s *Secure) Process(w http.ResponseWriter, r *http.Request) error {
 	}
 
 	// XSS Protection header.
-	if s.opt.BrowserXssFilter {
+	if s.opt.BrowserXSSFilter {
 		w.Header().Add(xssProtectionHeader, xssProtectionValue)
 	}
 
+	if len(s.opt.PublicKey) > 0 {
```

Comment:
Lets add the development and ssl check here:
`if len(s.opt.PublicKey) > 0 && isSSL && !s.opt.IsDevelopment {`